src/styles/ItemsList.css

```diff
@@ -0,0 +1,10 @@
+.shoppinglist ul {
```

Comment:
In our PR we have used hyphenated class names (eg. add-item) to be consistent with Nav.css where we used flexed-list. I guess it doesn't matter which format we use, as long as we are consistent, so which do we all prefer?

Reply:
You are totally right. I will change the `shoppinglist` to `shopping-list`. Is it ok to use `ul` afterwards or do you want me to do a className for everything we are going to change?

Reply:
I would even suggest using a more standard naming convention like [BEM](http://getbem.com/naming/) since we are nearing the last issue which requires us to style the app.

Reply:
Never worked with that before 😄 But, great suggestion, thank you! 👍 Since our code isn't that much and we have a stylesheet for every component I thought it would be more readable to just go with one className (ideally of the component itself) and than use the tags. So it would be easy to find what exactly is affected.

Reply:
I changed it to the first suggestion here to be consistent. But we still should consider Joseph's suggestion about BEM, I think.